packages/fxa-admin-panel/src/components/PageAccountSearch/Account/index.tsx

```diff
@@ -0,0 +1,390 @@
+                  {secondaryEmail.email}
+                </span>
+                <ResultBoolean
+                  isTruthy={secondaryEmail.isVerified}
+                  testId="secondary-verified"
+                />
+              </TableRowXHeader>
+            ))}
+          </TableXHeaders>
+        ) : (
+          <p className="result-none">
+            This account doesn't have any secondary emails.
+          </p>
+        )}
+
+        <EmailBounces {...{ emailBounces, uid, emails, onCleared }} />
+
+        <h3 className="header-lg">
+          2FA / TOTP (Time-Based One-Time Passwords)
+        </h3>
+        {totps && totps.length > 0 ? (
+          <TableXHeaders rowHeaders={['Created At', 'Enabled', 'Confirmed']}>
+            {totps.map((totp: TotpType) => (
+              <TableRowXHeader key={totp.createdAt}>
+                <td data-testid="totp-created-at">
+                  {getFormattedDate(totp.createdAt)}
+                </td>
+                <ResultBoolean isTruthy={totp.enabled} testId="totp-enabled" />
+                <ResultBoolean
+                  isTruthy={totp.verified}
+                  testId="totp-verified"
+                />
+              </TableRowXHeader>
+            ))}
+          </TableXHeaders>
+        ) : (
+          <p className="result-none">
+            This account hasn't started 2FA / TOTP setup.
+          </p>
+        )}
+
+        <h3 className="header-lg">Account Recovery Key</h3>
+        {recoveryKeys && recoveryKeys.length > 0 ? (
+          <>
+            {recoveryKeys.map((recoveryKey: RecoveryKeysType) => (
+              <TableYHeaders key={createdAt}>
+                <TableRowYHeader
+                  header="Created At"
+                  value={getFormattedDate(recoveryKey.createdAt)}
+                  testId="recovery-keys-created-at"
+                />
+                <TableRowYHeader
+                  header="Enabled"
+                  value={<ResultBoolean isTruthy={!!recoveryKey.enabled} />}
+                  testId="recovery-keys-enabled"
+                />
+                <TableRowYHeader
+                  header="Confirmed"
+                  value={<ResultBoolean isTruthy={!!recoveryKey.verifiedAt} />}
+                  testId="recovery-keys-verified"
+                />
+                <TableRowYHeader
+                  header="Confirmed At"
+                  value={
+                    recoveryKey.verifiedAt
+                      ? getFormattedDate(recoveryKey.verifiedAt)
+                      : HIDE_ROW
+                  }
+                />
+              </TableYHeaders>
+            ))}
+          </>
+        ) : (
+          <p className="result-none">
+            This account doesn't have an account recovery key created.
+          </p>
+        )}
+
+        <h3 className="header-lg">Subscriptions</h3>
+        {subscriptions && subscriptions.length > 0 ? (
+          <>
+            {subscriptions.map((subscription) => (
```

Comment:
Does this need a type for `subscription`?

Reply:
It is not needed in the sense that it is not required, but I think it would be good style to declare it. VSCODE knows it's type, which is MozSubscription.